01/app.js

```diff
@@ -1 +1,14 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const comentsNewest = document.querySelector(
+  '.comments__item.comments__item--newest'
+);
+if (comentsNewest) {
+  console.log(comentsNewest);
+  const commentLink = comentsNewest.querySelectorAll('[data-info]');
+  if (commentLink) {
```

Comment:
Drugi `if` jest niepotrzebny, bo używamy `querySelectoAll` ta metoda zwraca pustą tablice jak nic nie znajdzie więc to zawsze będzie prawda :) Pozatym nie będzie błędu bo `.length` zwróci 0. To całkiem inna sytuacja niż `querySelector`